### ISSUE TITLE
Question: New Release?

### ISSUE THREAD

**Issue by Artain** [matrix-org/matrix-synapse-ldap3#159]
When can we expect a new release? I can't use the plugin with the current synapse version because of the error mentioned in issue #133. The issue is fixed but no new release was created.

**Comment by reivilibre** [comment]
Hi there, thanks for raising this!

I can look at what it'd take to get another release published, but is there any chance you'd be happy to try out the updated module first and see if it works with a real LDAP server?
I haven't yet been able to test it against a real LDAP server because I don't have one; setting something up for development wasn't a five minute job either.

To do so, you can run
`pip install git+https://github.com/matrix-org/matrix-synapse-ldap3.git` inside your Synapse venv.
Update your config so it uses the `modules` section of the Synapse config file (rather than `password_providers`), and `LdapAuthProviderModule` (rather than `LdapAuthProvider`) — see https://github.com/matrix-org/matrix-synapse-ldap3/blob/main/README.rst#usage.
Then restart Synapse.

If something goes wrong you can put your configuration back (and see if that works),
otherwise if that's still broken, you can run this to downgrade:
```
pip uninstall matrix-synapse-ldap3
# Reinstall last release:
pip install matrix-synapse-ldap3
```

**Comment by Artain** [comment]
Thanks. I already did that on Sunday and it worked for me. I didn't got any issues after updating to the current version. But I have to say for now we only testet matrix and used only 2 Users from LDAP. I also knew exactly how these users were created and what attributes i needed so I didn't tested any edge cases or possible errors.

**Comment by reivilibre** [comment]
@Artain just to check, your config says `modules:` and `LdapAuthProviderModule` (the new style)?

**Comment by Artain** [comment]
This was the configuration I used, with redacted information of course. Also note I only tested ldap and no SSL encryption which isn't imported for me on this setup.

```
modules:
  - module: "ldap_auth_provider.LdapAuthProviderModule"
    config:
      enabled: true
      uri: "ldap://localhost:389"
      start_tls: false
      base: "ou=People,dc=DOMAIN,dc=TDL"
      attributes:
        uid: "uid"
        mail: "mail"
        name: "uid"
      bind_dn: "USER"
      bind_password: "Password"
      filter: "(memberOf=cn=matrix,ou=Group,dc=DOMAIN,dc=TDL)"
```

**Comment by reivilibre** [comment]
Perfect, thanks! I'll get a release put up shortly

**Comment by reivilibre** [comment]
I've released v0.2.0:
* https://github.com/matrix-org/matrix-synapse-ldap3/releases/tag/v0.2.0
* https://pypi.org/project/matrix-synapse-ldap3/0.2.0/

Hope that helps! Thanks.
